app/views/components/ilequery/response_buttons.scala.html

```diff
@@ -28,7 +28,7 @@
   <div>
       @govukButton(Button(content = Text(messages("ileQueryResponse.links.manageConsignment")), href = Some(routes.ChoiceController.displayPage().url)))
   </div>
-  <div class="govuk-body govuk-!-margin-bottom-9">
-      @link(messages("ileQueryResponse.links.findConsignment"), controllers.ileQuery.routes.IleQueryController.displayQueryForm)
+  <div>
```

Comment:
This section needs the space at the bottom to look right.  Was changed when the buttons moved from the bottom of the screen to the middle